cmd/serv.go

```diff
@@ -39,11 +39,6 @@ var CmdServ = cli.Command{
 	Description: `Serv provide access auth for repositories`,
 	Action:      runServ,
 	Flags: []cli.Flag{
-		cli.StringFlag{
```

Comment:
the authorized keyfile that is generated by gitea has each key start off with something like `command="/usr/bin/gitea serv key-1 --config='/var/lib/gitea/conf/app.ini'"` by removing the `--config` flag this will error out.

Reply:
Try it

Reply:
https://github.com/zeripath/gitea/blob/e17bb4fc032008a104e31fe89e2d315d96ce44e7/main.go#L106-L113

Reply:
Ah yes, clearly I should look at the whole PR before nitpitciking tiny parts of it. Sorry about that. Please disregard this comment.

Reply:
No problem